shell/platform/darwin/macos/framework/Headers/FlutterViewController.h

```diff
@@ -31,9 +31,19 @@ FLUTTER_DARWIN_EXPORT
 
 /**
  * The Flutter engine associated with this view controller.
+ *
+ * The engine is strongly referenced by the FlutterViewController, and weakly
+ * vice versa.
  */
 @property(nonatomic, nonnull, readonly) FlutterEngine* engine;
 
+/**
+ * The identifier for this view controller.
+ *
+ * The ID is assigned when the view controller is added to FlutterEngine.
```

Comment:
Maybe expand on this, like:
```suggestion
 * The ID is assigned when the view controller is assigned to the                             
 * FlutterEngine.viewController property.
```

Reply:
The mechanism is a little more complicated than this, so I decided to explain everything in FlutterViewController's doc. Can you take a look?